nautobot_version_control/__init__.py

```diff
@@ -138,8 +143,8 @@ def query_registry(model, registry):
 
 
 def is_versioned_model(model):
-    """
-    Determines whether a model's is under version control.
+    """Determines whether a model's is under version control.
+
```

Comment:
Ew, is Ruff enforcing this style? I'd hope that it would permit both styles, as both are valid Python docstrings.